bin/bst-proxy.ts

```diff
@@ -50,10 +54,11 @@ program
         console.log("Your URL for Alexa Skill configuration:");
         console.log(URLMangler.manglePipeToPath(Global.config().sourceID()));
         console.log("");
-        console.log("Your URL for viewing skill data:");
-        console.log(URLMangler.mangleNoPath(Global.config().sourceID(), Global.config().secretKey()));
-        console.log("Copy and paste this to your browser to your transaction history and summary data.");
-        console.log("");
+        // TODO: Re-add this lines once dashboard configuration is ready
+        // console.log("Your URL for viewing skill data:");
+        // console.log(URLMangler.mangleNoPath(Global.config().sourceID(), Global.config().secretKey()));
+        // console.log("Copy and paste this to your browser to your transaction history and summary data.");
```

Comment:
"Copy and paste this to your browser to view your transaction history and summary data." The word view is missing - I know this is commented out, but better to fix it now.